src/EnergyPlus/UnitarySystem.cc

```diff
@@ -5992,7 +5925,7 @@ namespace UnitarySystems {
             if (UtilityRoutines::SameString(loc_m_ISHundredPercentDOASDXCoil, "Yes")) {
```

Comment:
There is a `getYesNo` similar to `getEnumerationValue`.

Reply:
Not required field. Blank input is allowed so getYesNo won't work here.

Reply:
You just have to test for blank first. Not a big deal for now, just something to know for later.